koku/masu/test/database/test_aws_report_db_accessor.py

```diff
@@ -350,7 +350,7 @@ def test_delete_aws_hive_partition_by_month(self, mock_trino, mock_table_exist):
                 self.accessor.delete_hive_partition_by_month(table, self.ocp_provider_uuid, "2022", "01")
             mock_trino.assert_called()
             # Confirms that the error log would be logged on last attempt
-            self.assertEqual(mock_trino.call_args_list[-1].kwargs.get("attempts_left"), 0)
+            self.assertEqual(mock_trino.call_args_list[-1].kwargs.get("attempts_left"), None)
```

Comment:
This check can be removed since it's asserting something about a parameter that was removed. The important thing to check is that it was call the expected number of times.